draft-ietf-tls-external-psk-importer.md

```diff
@@ -183,40 +188,25 @@ PSKs imported for TLS 1.3 are distinct from those used in TLS 1.2, and thereby a
 cross-protocol collisions. Note that this does not preclude endpoints from using non-imported
 PSKs for TLS 1.2. Indeed, this is necessary for incremental deployment.
 
+# KDF Identifier Registry {#kdf-registry}
+
+The following table specifies KDF identities used in ImportedIdentity structures.
+
+| Value  | KDF         | Nh  | Reference    |
+|:-------|:------------|-----|:-------------|
+| 0x0000 | (reserved)  | N/A | N/A          |
+| 0x0001 | HKDF-SHA256 | 32  | {{?RFC5869}} |
+| 0x0002 | HKDF-SHA512 | 64  | {{?RFC5869}} |
```

Comment:
I assume that you plan to sort out which of this and HPKE will have the privilege of establishing this registry.  It doesn't make sense to duplicate effort.

Reply:
Correct :)